frontend-cert/js-projects/second-functions/user-stories.md

```diff
@@ -0,0 +1,20 @@
+1. You should define a function named `askQuestion` with a `question` parameter. The function should return a prompt with `question` as its parameter.
+
+2. You should define a function named `checkAnswer` with two parameters: `userAnswer` and `correctAnswer`. The fucntion should return the comparison of `userAnswer.toLowerCase()` and `correctAnswer.toLowerCase()`.
+
+3. You should define an arrow function named `runQuiz`. Step 4-7 should be created within the `runQuiz` function.
+
+4. You should create an array named `questions`. The array should have objects with `question` and `answer` properties. For example: `{ question: "What is the capital of France?", answer: "Paris" }`, `{ question: "What is 2 + 2?", answer: "4" },`, and so on. Create as many objects as you want.
```

Comment:
I think here we should put a limit and provide those questions so it is easier to test them.
What do you think about that?

Reply:
I agree. I've updated the user stories 👍